pkg/azclient/factory_gen.go

```diff
@@ -353,11 +353,12 @@ func (factory *ClientFactoryImpl) createAccountClient(subscription string) (acco
 		return nil, err
 	}
 	options.Cloud = factory.cloudConfig
-
 	if factory.armConfig != nil && strings.EqualFold(factory.armConfig.Cloud, utils.AzureStackCloudName) && !factory.armConfig.DisableAzureStackCloud {
 		options.ClientOptions.APIVersion = accountclient.AzureStackCloudAPIVersion
 	}
-
+	if !strings.EqualFold(options.Cloud.ActiveDirectoryAuthorityHost, cloud.AzurePublic.ActiveDirectoryAuthorityHost) {
```

Comment:
should be else if here, so that it won't overwrite L357

Reply:
switched the order. azurestack config will override non-public cloud config. Thanks!